tests/test-umockdev-vala.vala

```diff
@@ -268,17 +268,14 @@ E: SUBSYSTEM=usb
 
   /* no ioctl tree loaded */
   var ci = Ioctl.usbdevfs_connectinfo();
-  assert_cmpint (Posix.ioctl (fd, Ioctl.USBDEVFS_CONNECTINFO, ref ci), CompareOperator.EQ, -1);
-  // usually ENOTTY, but seem to be EINVAL
-  assert_cmpint (Posix.errno, CompareOperator.GE, 22);
-  errno = 0;
```

Comment:
This should go back. This test runs the actual ioctl, and checking the error code gives some additional confidence that this happens. It is of course more robust to also reset errno to 0 before, just like in the next check.